packages/superset-ui-time-format/src/utils/createDate.ts

```diff
@@ -0,0 +1,13 @@
+export default function createDate({
+  year,
+  month = 0,
+  date = 1,
+  useLocalTime = false,
+}: {
+  year: number;
+  month?: number;
+  date?: number;
+  useLocalTime?: boolean;
+}): Date {
+  return useLocalTime ? new Date(year, month, date) : new Date(Date.UTC(year, month, date));
+}
```

Comment:
Will this ever need hour minutes seconds?

Reply:
This was used internally for dates at the moment. I could rename to `createTime` and add support for hour, minutes, seconds, milliseconds.